appinfo/info.xml

```diff
@@ -35,7 +35,7 @@
 	<screenshot>https://raw.githubusercontent.com/nextcloud/calendar/main/screenshots/week_sidebar.png</screenshot>
 	<dependencies>
 		<php min-version="7.4" max-version="8.2" />
-		<nextcloud min-version="25" max-version="27" />
+		<nextcloud min-version="25" max-version="28" />
```

Comment:
nit: not part of the chore, this is a feature. this should probably come separately so it's included in the changelog of v4.5